ietf/doc/views_doc.py

```diff
@@ -870,6 +870,12 @@ def document_main(request, name, rev=None, document_html=False):
                     and doc.group.features.has_nonsession_materials
                     and doc.type_id in doc.group.features.material_types
             )
+
+        session_statusid = None
+        if doc.session_set.count() == 1:
+            if doc.session_set.get().schedulingevent_set.exists():
```

Comment:
Based on the test failure, this needs to handle the case that `doc` is a `DocHistory`